<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>811</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{
background-color:qlineargradient(spread:pad, x1:0.419, y1:0.767227, x2:1, y2:0.534, stop:0 rgba(147, 165, 207, 255), stop:1 rgba(228, 239, 233, 255))
}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>20</y>
      <width>421</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Sans Serif Collection&quot;;</string>
    </property>
    <property name="text">
     <string>Enter the details of Hod</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="hodName">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>451</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:17px;
background-color:rgba(0,0,0,0);
border:3px solid rgb(139, 139, 139);
font: 18pt &quot;Sans Serif Collection&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Sans Serif Collection&quot;;
</string>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Sans Serif Collection&quot;;
</string>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="hodEmail">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>451</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:17px;
background-color:rgba(0,0,0,0);
border:3px solid rgb(139, 139, 139);
font: 18pt &quot;Sans Serif Collection&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>340</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Sans Serif Collection&quot;;
</string>
    </property>
    <property name="text">
     <string>Number:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="hodNumber">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>340</y>
      <width>451</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:17px;
background-color:rgba(0,0,0,0);
border:3px solid rgb(139, 139, 139);
font: 18pt &quot;Sans Serif Collection&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="hodBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>480</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:8px;
border:3px solid rgb(139, 139, 139);
background-color:rgba(0,0,0,0);
font: 22pt &quot;Sans Serif Collection&quot;;</string>
    </property>
    <property name="text">
     <string>Appoint Hod</string>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>420</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Sans Serif Collection&quot;;
color:red;
background-color: rgba(0,0,0,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
